internal/operators/metallb/manifest.go

```diff
@@ -0,0 +1,229 @@
+package metallb
```

Comment:
We have a new mechanism for defining manifests for operators which is more robust and less error prone.  Please see how it is used in one of the latest added operators, for example `nodehealthcheck`